keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -510,6 +514,7 @@ type Namespaces struct {
 	// Labels are set only when scope is "SameLabels".
 	Labels      []string
 	// Selector is only set when scope is "Selector".
+	// Namespaces.Selector has the same effect as NamespaceSelector.
```

Comment:
we can keep it for now; but i wonder if we need this strategy now or let users use NamespaceSelector